examples/svelte-kitchen-sink/svelte-format/__snapshots__/argstable.stories.storyshot

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Storyshots Args Table (Svelte) ArgsTable 1`] = `
+<section
+  class="storybook-snapshot-container"
+>
+   
+  <pre>
+    {
+  "string": "string",
+  "number": 0,
+  "unionstr": "a",
+  "unionnumeric": 1,
+  "required": ""
+}
+  </pre>
+   
+  <div />
+   
+  <div />
+   
+   
+  
+  
```

Comment:
Why so much whitespace in some of these snapshots?

Reply:
I have the same problem in 6.1 in the projects I manage. svelte keeps spaces in template, so snapshots generated by storyshoots has a lot of spaces. If you change just indentation in a svelte component, the snapshots failed.

Maybe I can try to implements a new snapshots serializer for svelte, but it will break every already generated snapshots